pylint/lint/utils.py

```diff
@@ -99,3 +99,16 @@ def fix_import_path(args: Sequence[str]) -> Iterator[None]:
         yield
     finally:
         sys.path[:] = original
+
+
+def _is_relative_to(self: Path, *other: Path) -> bool:
```

Comment:
```suggestion
def pathlib_relative_to_is_callable(self: Path, *other: Path) -> bool:
```
? I don't think I understand what this function does. Shouldn't the value be returned too ?

Reply:
It's just a line for line copy of the `stdlib` function. We don't really need to return anything else?